server/app/app.py

```diff
@@ -136,24 +135,29 @@ def run_scanpy(args):
         DATASET_TITLE=title,
         CXG_API_BASE=api_base
     )
-    if not args.debug:
+
+    if not args.verbose:
         log = logging.getLogger('werkzeug')
         log.setLevel(logging.ERROR)
     from .scanpy_engine.scanpy_engine import ScanpyEngine
-    print(f"Loading data from {args.data}")
+    print(f"Loading data from {args.data} (this may take a while for large datasets)")
```

Comment:
I would just say "this may take a while", as some users have slow computers and "large datasets" is a very relative thing....

Reply:
👍